### ISSUE TITLE
T2T CHM13 Compatibility 

### ISSUE THREAD

**Issue by nataliagd12** [cortes-ciriano-lab/savana#58]
Hello, 

I have successfully run Savana with hg38 as a reference genome. I am interested in using the T2T reference genome for some analysis, is the latest version compatible with this ref genome? 

Thank you!

**Comment by nataliagd12** [comment]
Thank you for your swift response! I have aligned the input BAMs to T2T (specifically the NCBI RefSeq assembly) but tried running without a phased VCF and got an error saying "ValueError: not enough values to unpack (expected 2, got 1)" around the getting breakpoints step. I will try running with phased VCF and see if that resolves the issue.

**Comment by nataliagd12** [comment]
I have tried running with the phased VCF and am still gettin the same error. Anyone else having this issue?

**Comment by helrick** [comment]
Hi there, sorry to hear you're having issues using the T2T reference! Would you be able to supply the exact command and error message you're getting? It could be that the NCBI RefSeq reference has characters which aren't handled properly. Are you able to share a subset of your aligned BAM?

**Comment by nataliagd12** [comment]
![Image](https://github.com/user-attachments/assets/83cf1a02-7949-4ad5-bb5d-d141a61fce66)
![Image](https://github.com/user-attachments/assets/f52204dc-8e9f-4ff7-a8d3-d359f73220b8)

![Image](https://github.com/user-attachments/assets/e85cf7e9-88c5-4aa0-bf53-172ea77c795a)

Here are images of my command, part of the error message, and part of the read header. One idea is that is the chromosomes in the NCBI refseq reference are "NC_060925.1" instead of "chr1"... does savana need chr1 format?

**Comment by helrick** [comment]
Hi there, would you be able to share the BAM file itself? Or potentially a subset version of it? One more question, what aligner did you use to align to the reference genome?

**Comment by nataliagd12** [comment]
pbmm2 was used for alignment. I'm working on getting a subset of the BAM that I could share but is there a specific field in the reads or headers I can start looking through? Another thought is if i need to enable the single breakend option?

**Comment by helrick** [comment]
Hi there, SAVANA has been tested on ONT and PacBio HiFi reads aligned with minimap2 and winnowmap only, we currently don't support using pbmm2 as an aligner. This could be the source of the issue. Are you able to realign your BAM to T2T using minimap2?

**Comment by nataliagd12** [comment]
Yes the previous run I did used pbmm2 as well and worked all right. My sample is currently aligned to the CHM13 T2T reference genome so I'm a little confused by your last suggestion. 

I am working with classified patient data so unfortunately I am not able to share even a subset of the BAM but I am working on recreating the error with COLO829 pacbio data which is able to be shared.

**Comment by helrick** [comment]
Hi there, apologies for the misunderstanding. I was merely suggesting that you try with the same FASTA file for the T2T reference that we have used successfully ourselves, rather that the NCBI RefSeq version that you mentioned you were using. The file I provided a link to is from the T2T consortium itself ([see here, the GitHub referenced in their publication](https://github.com/marbl/CHM13)) and denotes its chromosomes in the format 'chr1'. As you noted, the RefSeq version of the T2T reference instead denotes chromosome 1 as "NC_060925.1".

To be more specific, if we look at the first few lines of the FASTA file I shared a link to above, we can see that the first line contains:
```
> cat chm13v2.0.fa | head -n 5
>chr1 CP068277.2 Homo sapiens isolate CHM13 chromosome 1
Caccctaaaccctaacccctaaccctaaccctaaccctaaccctaaccctaacccctaaaccctaaccctaaccctaacc
ctaaccctaaccctaaccctaaccctaaccctaaccctaaccctaaccctaaccctaaccctaaccctaaccctaaccct
aaccctaaccctaacccctaaccctaaccctaaccctaaccctaaccctaaccctaaccctaaccctaaccctaacccta
accctaaccctaaccctaaccctaaccctaaccctaaccctaacccaaccctaaccctaaccctaaccctaaccctaacc
```

In comparison, the RefSeq FASTA file looks like:
```
> zcat GCF_009914755.1_T2T-CHM13v2.0_genomic.fna.gz | head -n 5
>NC_060925.1 Homo sapiens isolate CHM13 chromosome 1, alternate assembly T2T-CHM13v2.0
CACCCtaaaccctaacccctaaccctaaccctaaccctaaccctaaccctaacccctaaaccctaaccctaaccctaacc
ctaaccctaaccctaaccctaaccctaaccctaaccctaaccctaaccctaaccctaaccctaaccctaaccctaaccct
aaccctaaccctaacccctaaccctaaccctaaccctaaccctaaccctaaccctaaccctaaccctaaccctaacccta
accctaaccctaaccctaaccctaaccctaaccctaaccctaacccaaccctaaccctaaccctaaccctaaccctaacc
```

Again, I am not sure if this is the source of the issue, but in the meantime while you're working on generating a share-able BAM, you are welcome to test aligning to the reference I provided to see if it resolves the issue.
